rio_cogeo/scripts/cli.py

```diff
@@ -135,7 +136,10 @@ def cogeo():
     show_default=True,
 )
 @click.option(
-    "--web-optimized", "-w", is_flag=True, help="Create COGEO optimized for Web."
+    "--web-optimized",
+    "-w",
+    is_flag=True,
+    help="Create COGEO optimized for Web. Deprecated: use directly --tms",
```

Comment:
I would maybe not deprecated `web-optimized` in the CLI, IMO it's nice not to have to select any other option (98% of the web-optimized user will want to use the WebMercator TMS)

Reply:
Yes, you're right. I will remove the deprecate part.